src/components/message/Message.tsx

```diff
@@ -23,11 +14,12 @@ export enum EMessageType {
 // and we don't want to accidentally import "Message" instead of "message" on our app
 
 export interface IMessage {
-  title?: string;
   type: string;
   content: string;
   id: number;
   duration: number;
+  icon?: React.ReactNode;
+  action?: React.ReactNode;
```

Comment:
@steiningermona action interface accepts any node element here. This should be forced to be an interface with 2 properties:

```
{
 actionLabel: string;
 actionClick: () => void;
}
```
..or similar, to prevent user component to pass anything that would break the interface. Unless there is any other objection to that.